Makefile

```diff
@@ -41,3 +41,7 @@ fmt:
 	$(NPM) run format
 
 # vim: ft=make
+
+.PHONY: file
+file:
```

Comment:
Without `npm install` this will fail on fresh clones of the folder.

I dont think you need the template generator to be a separate npm project, but @johanstokking will know better than me. If you consolidate it in the root folder, then `make deps` will already handle this with `npm install`. Otherwise, you need to add something in the makefile to cwd to your project and run `npm install`